src/ui/MessageInput/index.jsx

```diff
@@ -329,6 +329,11 @@ const MessageInput = React.forwardRef((props, ref) => {
         } else if (node.nodeType === NodeTypes.ElementNode && node.nodeName === NodeNames.Br) {
           messageText += '\n';
           mentionTemplate += '\n';
+        } else if (node?.nodeType === NodeTypes.ElementNode && node?.nodeName === NodeNames.Div) {
```

Comment:
Q. Just curious, how do we know if it's Safari env or not? 
>node?.nodeName === NodeNames.Div

^^ This one?

Reply:
so, div shouldnt arrive otherwise into MessageInput(checked fire fox and chrome)

Only other way dom element arrive is through paste(hope I am not wrong) -> we extract text for those cases
You think I should put strong check on browser type?